app/views/admin/conference_domains/show.haml

```diff
@@ -17,14 +17,15 @@
           %td
             = @conference.custom_domain
           %td
-            - ck_domain = ConferenceDomainsService.new(conference: @conference).check_custom_domain
-            - if ck_domain == true
-              %span.glyphicon.glyphicon-ok
-            - elsif ck_domain == false
-              %span.glyphicon.glyphicon-remove
+            - if ENV['OSEM_HOSTNAME'].present?
+              - ck_domain = ConferenceDomainsService.new(@conference).check_custom_domain
+              - if ck_domain
+                %span.glyphicon.glyphicon-ok
+              - else
+                %span.glyphicon.glyphicon-remove
             - else
               %i
-                = ck_domain
+                feature disabled
```

Comment:
would it make sense maybe to tell the user WHY the feature is disabled, and how to enable it?